add-on/chrome/bootstrap.js

```diff
@@ -1431,6 +1473,11 @@ function shutdown(data, reason) {
   // Stop waiting for browser windows to open.
   wm.removeListener(WindowListener);
 
+  // unregister about: handlers
+  AboutLoop.conversation.unregister(); // eslint-disable-line no-undef
+  AboutLoop.panel.unregister(); // eslint-disable-line no-undef
+  AboutLoop.toc.unregister(); // eslint-disable-line no-undef
```

Comment:
Hmm, I think this was mentioned in Mark's original review comments.  Do we still need to go through and address his original comments now that we've unforked?